source/_components/sensor.etherrain.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "EtherRain Sensor"
+description: "Monitor the state of your EtherRain"
+date: 2017-08-09 06:16
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: etherrain.jpg
+ha_category: Sensor
+ha_release: 0.31
```

Comment:
HA release will most like need to be updated to 0.52  since 0.51 is rather imminent.